napari/layers/tracks/tracks.py

```diff
@@ -0,0 +1,489 @@
+# from napari.layers.base.base import Layer
+# from napari.utils.events import Event
+# from napari.utils.colormaps import AVAILABLE_COLORMAPS
+
+from typing import Union
+
+import numpy as np
+
+from ...utils.colormaps import AVAILABLE_COLORMAPS
+from ...utils.events import Event
+from ..base import Layer
+from ._track_utils import TrackManager, get_track_dimensionality
+
+ALLOW_ND_SLICING = False
+
+
+class Tracks(Layer):
+    """ Tracks
+
+    A Tracks layer for overlaying trajectories on image data.
+
+    Parameters
+    ----------
+    data : list
+        list of (NxD) arrays of the format: time, x, y, (z), ....
```

Comment:
Our data is generally ..., T, Z, Y, X for an nD image layer or points layer. I havn't looked in code yet, but is the ordering here consistent with those other layers?

Reply:
Yes - even though the incoming data is T,X,Y,(Z), the layer reorders the axes for display in the `_pad_display_data` function, to be consistent with the nD image/points layer data.